app/src/main/java/com/pileproject/drive/execution/ExecutionThread.java

```diff
@@ -158,155 +117,56 @@ private void finalizeExecution() {
         }
     }
 
-    // Wait on the millisecond time scale
+    // wait in milliseconds
     private void waitMillSec(int milli) {
         try {
-            Thread.sleep(milli); // Sleep for milli sec
+            Thread.sleep(milli); // sleep for milli sec
         } catch (InterruptedException e) {
```

Comment:
In this case, InterruptedException often occurs while a program execution to stop the program immediately by a user action, 'stop button'. So, as for this method, the exception can be ignored. But I will fix other methods, thanks :)